packit_service/worker/events/event.py

```diff
@@ -492,3 +492,21 @@ def get_all_build_targets_by_status(
             models=CoprBuildTargetModel.get_all_by_commit(commit_sha=self.commit_sha),
             statuses_to_filter_with=statuses_to_filter_with,
         )
+
+
+class AbstractResultEvent(AbstractForgeIndependentEvent):
+    """
+    This class is used only as an Abstract for result events to
+    allow Steve properly filter jobs with manual trigger.
+    """
+
+    def __init__(
+        self,
+        created_at: Union[int, float, str] = None,
+        project_url=None,
+        pr_id: Optional[int] = None,
+        actor: Optional[str] = None,
+    ):
+        super().__init__(
+            created_at=created_at, project_url=project_url, pr_id=pr_id, actor=actor
+        )
```

Comment:
```suggestion
    pass
```

This should work fine if we don't do anything there.

Reply:
and when changing this, could you please squash your commits?

Reply:
good, didn't know about that :) Pushed